<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>350</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f5f5f5;
}</string>
  </property>
  <property name="windowTitle">
   <string>BMI Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    background-color: #e8f4fc;
    border: 2px solid #b8d4eb;
    border-radius: 5px;
    margin-top: 1ex;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Units</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="radio_metric">
         <property name="styleSheet">
          <string notr="true">color: #2c3e50;</string>
         </property>
         <property name="text">
          <string>Metric (kg, m)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_imperial">
         <property name="styleSheet">
          <string notr="true">color: #2c3e50;</string>
         </property>
         <property name="text">
          <string>Imperial (lbs, inches)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_weight">
      <property name="styleSheet">
       <string notr="true">color: #34495e;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Weight:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_weight">
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff;
border: 1px solid #bdc3c7;
border-radius: 3px;
padding: 5px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_height">
      <property name="styleSheet">
       <string notr="true">color: #34495e;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_height">
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff;
border: 1px solid #bdc3c7;
border-radius: 3px;
padding: 5px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_calculate">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #27ae60;
    color: white;
    border: none;
    padding: 8px;
    border-radius: 4px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #219a52;
}</string>
      </property>
      <property name="text">
       <string>Calculate BMI</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_result">
      <property name="styleSheet">
       <string notr="true">background-color: #e8f5e9;
color: #2ecc71;
border: 1px solid #2ecc71;
border-radius: 4px;
padding: 8px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>BMI: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_status">
      <property name="styleSheet">
       <string notr="true">background-color: #f0f4f7;
color: #3498db;
border: 1px solid #3498db;
border-radius: 4px;
padding: 8px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Status: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ecf0f1;
color: #2c3e50;</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QAction" name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QAction" name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QAction" name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>